uppsala-universitet-institutionen-for-biologisk-grundutbildning.csl

```diff
@@ -14,29 +14,32 @@
     <category citation-format="author-date"/>
     <category field="generic-base"/>
     <summary>Style for Swedish and English texts done at the Biology Education Centre (Institutionen för biologisk grundutbildning) at Uppsala University, Sweden.</summary>
-    <updated>2015-10-22T09:00:00+00:00</updated>
+    <updated>2016-05-16T09:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+
   <locale>
     <terms>
       <term name="et-al">et al.</term>
     </terms>
   </locale>
+
   <locale xml:lang="en-GB">
     <date form="text">
-      <date-part name="day" suffix=" "/>
-      <date-part name="month" suffix=" "/>
-      <date-part name="year"/>
+      <date-part name="day" suffix=" " />
```

Comment:
not sure where these come from, but could you remove the trailing spaces here (i.e. the space _after_ the last ") and below in row 40-42?